### ISSUE TITLE
Project Directed Funding

### ISSUE THREAD

**Issue by jorydotcom** [openjs-foundation/cross-project-council#90]
update from @mylesborins:

the OpenJS Foundation now has [an official process for Project Directed Funding.](https://openjsf.org/about/project-funding-opportunities/) This is specifically focused on member organizations directing funding to support a specific project in the foundation. We are still working on how to best support the variety of crowd-funding platforms that projects may want to utilize, and will have an update related to that in the future

---

On today's bootstrap call we indirectly brought up the question of whether and how a company could direct funds to specific projects via the Foundation. I took the action item to file the discussion issue. As I understand it, neither foundation has a clear way of addressing this need (JSF attempted to handle this with a 'domains' concept, but I don't know if this was ever tested). 

> Do organizations want to be able to direct funds explicitly to certain projects? If so, how?

I think this will be a dynamic problem to tackle, so I'm not recommending that we attempt to solve it prior to executing the merge documents. It also has potential implications for members, projects and the foundation's business model, so I'd like to suggest we form a representative committee to explore the issue once the new $Foundation's activities are underway. 

Thoughts?

**Comment by mhdawson** [comment]
I agree with @mcollina that we should talk about some of the potential models so that we make sure we could accommodate those in the future.

One would be:
  * $fee to join the foundation
  * after joining foundation, $fee to join/support specific project.
  * can only sponsor specific project after joining foundation itselve

Another might be
  * $ to join the foundation
   * member can direct x% of their membership fee to specific project(s).

Another consideration is other funding sources (which I think @boneskull was mentioning) like patreon etc. and how that fits with the foundation.

**Comment by knolleary** [comment]
On the topic of other sources of funds is ad revenue. We were approached by an ad network to host developer-centric ads on our project site. Ultimately we declined as it wasn't clear where that sat in terms of JSF policy - would any revenue feed back into the JSF at large, or into a dedicated pot for our project. I don't have a particular desire to host ads on the project site, but I have seen other JSF projects do.

**Comment by MylesBorins** [comment]
So alternative take. If the CPC is chartered with budgeting... and is trusted by projects to do so... would directed funding not undermine that charter?

**Comment by joesepi** [comment]
Thinking outloud here as well, but what about the concept of a "sponsor." For example, if project x had need y and got company z to "sponsor" the solution to need y. Maybe a direct dollars to need from a company could get around some of the complexities you are referring to, @MylesBorins?

**Comment by jorydotcom** [comment]
Summarizing the discussion we had in today's merge meeting on this issue: Andy (LF Counsel) advised that this topic (earmarking funds for specific projects) would not be prohibited by our bylaws as as currently drafted; leaving it out gives us the most flexibility for now.   

I'm removing the bootstrap-agenda label as I don't think we need to spend more time on this in the merge committee specifically; I think it would be good to transfer this issue to the CPC as soon as it is formed. By all means though, feel free to add it back if you prefer.

**Comment by joesepi** [comment]
Under active discussion at Board level per @brianwarner

**Comment by MylesBorins** [comment]
I've sent an email to the board to get discussion moving

**Comment by tobie** [comment]
Here are a few ways projects might want to use earmarked funds. Some of them may have complex implications (e.g. paying wages to someone in Iran) and might not always be possible. It would be great to be able to be upfront about the limitations:
 
- Cover travel expenses of contributors.
- Pay for the work of some or all contributors, where contributors can be in pretty much any geographical location and might not have an independant status (can't invoice and need to be paid a wage instead).
- Fund external open source projects a project relies on (e.g. through OpenCollective).
- Operating ongoing expenses (servers, CI, software licenses, etc.).
- Third-party vendor for specific services (e.g. security or privacy audit, design, etc.).
- Swag for contributors. 
- Organizing events in locations across the world including related expenses (marketing, etc.)

Are you OK with those staying here or would you rather I opened another issue for them?

Another revenue source might be selling tickets for events. I'm pretty sure the LF knowns how to handle that, but it would be good to have confirmation the OpenJS foundation can do it too.

**Comment by jorydotcom** [comment]
@tobie I think this is a good spot to capture these points - thank you for adding them!

**Comment by tobie** [comment]
I see that this has now been labeled as requiring board approval. Are there any publicly available updates on this? This is something that's critical for AMP moving forward.

**Comment by joesepi** [comment]
Very active and ongoing topic of discussion at the board.

**Comment by mhdawson** [comment]
Policy for one aspect was shared in yesterdays public board meeting.  Text will be on openJS website soon under the member benefits section.

**Comment by mhdawson** [comment]
This is no longer waiting on the board, the guidance was shared.

**Comment by mhdawson** [comment]
For reference the link to the guidance: https://openjsf.org/about/project-funding-opportunities/

@jorydotcom can this issue be closed now?

**Comment by mcollina** [comment]
Closing this, feel free to reopen in case it's wrong.
